llama_stack/apis/uploads/uploads.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from typing import Optional, Protocol, runtime_checkable
+
+from llama_models.schema_utils import json_schema_type, webmethod
+from pydantic import BaseModel
+
+from llama_stack.apis.files import FileResponse
+from llama_stack.providers.utils.telemetry.trace_protocol import trace_protocol
+
+
+@json_schema_type
+class UploadResponse(BaseModel):
+    id: str
+    url: str
+
+
+@runtime_checkable
+@trace_protocol
+class Uploads(Protocol):
+    @webmethod(route="/uploads", method="POST")
+    async def init_upload(
+        self,
+        bucket: str,
+        key: str,
+        mime_type: str,
+    ) -> Optional[UploadResponse]: ...
+
+    @webmethod(route="/uploads/{upload_id}", method="POST")
+    async def upload_content(
+        self,
+        upload_id: str,
+    ) -> Optional[FileResponse]: ...
```

Comment:
This one should have request body generated as: 

```
requestBody:
  content:
    application/octet-stream:
      schema:
        type: string
        format: binary
```

Reply:
Solved the problem in generator by looking at the request body params and checking if the type is bytes to mark content type as octet-stream.